### DISCUSSION TITLE
osu!lazer not showing

### DISCUSSION THREAD

**Discussion by kurox313** [ppy/osu#16220] [Q&A]
I wanted to play osu(lazer) so I downloaded it from Github and install it after that the game worked fine after 4hr passed I clicked osu!(lazer) and it does not show the windows I looked in task manager and looked at the background processes and I see osu was there and I have no idea how osu is there so can some body tell me?

**Comment by peppy** [comment]
Do you still have it open? If so, can you right click it in task manager and choose "Create process dump" then email that to me (pe@ppy.sh)?

**Reply by kurox313** [reply]
ill try that thanks

**Comment by bdach** [comment]
Also if you have an nvidia card and shadowplay enabled then please try disabling that.

**Reply by kurox313** [reply]
alright

**Reply by kurox313** [reply]
wait how do i do that

**Reply by GH-vanny** [reply]
Alt + Z

Instant Replay > Turn Off
